packages/gitbook/src/app/sites/static/[mode]/[siteURL]/[siteData]/~gitbook/embed/script.js/route.ts

```diff
@@ -7,6 +7,13 @@ import type { NextRequest } from 'next/server';
 
 export const dynamic = 'force-static';
 
+const EMBEDDABLE_RESPONSE_HEADERS = {
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Methods': 'GET, OPTIONS',
+    'Cross-Origin-Resource-Policy': 'cross-origin',
+    'Cache-Control': 'public, max-age=86400, stale-while-revalidate=604800',
```

Comment:
The `max-age` could be a bit bad if we make some changes, we don't have control over it at all
```suggestion
    'Cache-Control': 'public, s-maxage=86400, stale-while-revalidate=604800',
```